### ISSUE TITLE
Is there any way to retain i tags in cleaned html ? 

### ISSUE THREAD

**Issue by pranjalkoshti** [jodit/jodit-react#164]
While editing an html, jodit is removing i tag completely, even tried to replace it with em but no luck. Is there any way to keep i tag as it is or to bypass cleanHtml flow itself ? 
Thanks.

**Comment by xdan** [comment]
Yes — that's cleanHTML.replaceOldTags. By default it maps i→em and b→strong on paste/cleanup; set cleanHTML: { replaceOldTags: false } and your <i> tags survive. Closing as answered.
